src/Input/Input.js

```diff
@@ -89,6 +129,22 @@ module.exports = kind(
 	},
 
 	/**
+	* @private
+	*/
+	components: [
+		{name: 'TooltipDecorator', kind: TooltipDecorator, autoShow: false, components: [
```

Comment:
Also, curious if we actually need a TooltipDecorator in this case. It seems like it could be problematic if the decorator needs to wrap our activator, which is the input in this case...